text_extensions_for_pandas/array/token_span.py

```diff
@@ -218,6 +218,15 @@ def name(self) -> str:
         """:return: A string representation of the dtype."""
         return "TokenSpanDtype"
 
+    @property
+    def na_value(self) -> object:
+        """
+        See docstring in `ExtensionDType` class in `pandas/core/dtypes/base.py`
+        for information about this method.
+        """
+        empty_span_array = SpanArray(None, [], [])
+        return TokenSpan(empty_span_array, Span.NULL_OFFSET_VALUE, Span.NULL_OFFSET_VALUE)
```

Comment:
As a followup, it would be good to use these values in an array to check that they don't cause problems.